### PR TITLE
[MIG] sale_order_invoicing_finished_task: Migration to 11.0

### PR DESCRIPTION
**Pull request by carlosdauden**
@Tecnativa

### CODE REVIEW

**Comment by carlosdauden** [comment]
CC @leemannd @gurneyalex

File: sale_order_invoicing_finished_task/models/project.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        if 'invoiceable' in vals:
+            self.mapped('sale_line_id')._analytic_compute_delivered_quantity()
+        return res
+
+    @api.model
+    def create(self, vals):
+        SOLine = self.env['sale.order.line']
+        so_line = SOLine.browse(vals.get('sale_line_id'))
+        # We don't want to add a project.task to an already invoiced line
+        if so_line and so_line.state in ('done', 'cancel'):
```
Comment (pedrobaeza) [inline sale_order_invoicing_finished_task/models/project.py]: This should check `invoice_status`, not `state`

File: sale_order_invoicing_finished_task/models/project.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            raise ValidationError(_('You cannot add a task to and invoiced '
```
Comment (pedrobaeza) [inline sale_order_invoicing_finished_task/models/project.py]: s/to and invoiced/to an invoiced

File: sale_order_invoicing_finished_task/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Copyright 2017 Sergio Teruel <sergio.teruel@tecnativa.com>
+# Copyright 2017-2018 Carlos Dauden <carlos.dauden@tecnativa.com>
+# Copyright 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "Sale Order Invoicing Finished Task",
+    "summary": "Control invoice order lines if his task has been finished",
```
Comment (pedrobaeza) [inline sale_order_invoicing_finished_task/__manifest__.py]: s/his/its related

File: sale_order_invoicing_finished_task/models/product.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    invoicing_finished_task = fields.Boolean(
+        string='Invoicing control by task',
+        help='Invoice the order lines only when the task is set invoiceable',
```
Comment (pedrobaeza) [inline sale_order_invoicing_finished_task/models/product.py]: s/task is set invoiceable/task is set to invoiceable

File: sale_order_invoicing_finished_task/static/description/icon.svg [hunk 5]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (pedrobaeza) [inline sale_order_invoicing_finished_task/static/description/icon.svg]: Don't include this SVG

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by carlosdauden** [comment]
Changes done

File: sale_order_invoicing_finished_task/__manifest__.py [hunk 6]
```diff
@@ -5,7 +5,8 @@
 
 {
     "name": "Sale Order Invoicing Finished Task",
-    "summary": "Control invoice order lines if his task has been finished",
+    "summary": "Control invoice order lines if its related task has been "
```
Comment (pedrobaeza) [inline sale_order_invoicing_finished_task/__manifest__.py]: Sorry, I told you bad. It's `their related`, not `its related`. Please change this and squash directly the commits.

**Comment by carlosdauden** [comment]
Changes and squash done

**Review by pedrobaeza — APPROVED** [review]

**Review by rafaelbn — APPROVED** [review]
Tested in runbot :+1: 

Thanks @carlosdauden

**Pull request merged**